### ISSUE TITLE
[Chips] Support custom hit areas

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3513]
### Overview
Chips are only 36 points high (by default) and therefore fail to meet accessibility guidelines (48 x 48 point target) for touch target size. 

### Version number
54.11.1
